160_Intersection_of_Two_Linked_Lists.py

```diff
@@ -0,0 +1,22 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+# time O(m+n) m:length of headA, n:length of headB
+# space O(m+n)
+class Solution:
+    def getIntersectionNode(self, headA: ListNode, headB: ListNode) -> ListNode:
+        if not headA or not headB:
+            return None
```

Comment:
Insert a blank line after the return statement

Reply:
- I insert a blank line after the return statement